packages/ui/apis/axiosInstance.ts

```diff
@@ -0,0 +1,61 @@
+import axios, { AxiosInstance, AxiosRequestConfig } from 'axios';
+
+import {
+  onErrorRequest,
+  onErrorResponse,
+  onRequest,
+  onResponse,
+} from './axiosInterceptor';
+
+const BASE_URL = '/example';
+
+class AxiosService {
+  private static instance: AxiosInstance;
+
+  private constructor() {}
+
+  private static getInstance(): AxiosInstance {
+    if (!AxiosService.instance) {
+      AxiosService.instance = axios.create({ baseURL: BASE_URL });
+      AxiosService.addInterceptors(AxiosService.instance);
+    }
+    return AxiosService.instance;
+  }
+
+  private static addInterceptors(instance: AxiosInstance) {
+    instance.interceptors.request.use(onRequest, onErrorRequest);
+    instance.interceptors.response.use(onResponse, onErrorResponse);
+  }
+
+  public static async get<T, U>(
```

Comment:
`Generic` 을 구체적으로 명시해주는 것도 클린코드 관점에서 좋다고 알고 있습니다!
```suggestion
  public static async get<Reponse, Request>(
```

Reply:
좋은 제안인 것 같습니다